app/routes/routes.js

```diff
@@ -2,7 +2,10 @@ const express = require('express');
 const router = express.Router();
 
 const member = require('./member.route.js');
+const auth = require('./auth.route.js');
```

Comment:
Codacy has a fix for the issue: [Replace `'./auth.route.js'` with `"./auth.route.js"`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9684192)
```suggestion
const auth = require("./auth.route.js");
```